apps/public/src/app/profile/wallet/pages/wallet/wallet.html

```
@@ -0,0 +1,81 @@
+    matIconButton
+    type="button"
+    aria-label="Voltar para a lista de cartões"
+    [disabled]="walletView() === 'closing'"
+    (click)="returnToCardList()">
+    <mat-icon>arrow_back</mat-icon>
+  </button>
+  }
+  <span>Carteira</span>
+  <span class="global-toolbar-spacer"></span>
+  @if (walletView() === 'list') {
+  <button matIconButton aria-label="Imprimir crachá" matTooltip="Imprimir crachá" (click)="print()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)